### PR TITLE
🚤 Switch message export to use XLSLite

### PR DESCRIPTION
**Pull request by norkans7**
Addresses https://github.com/rapidpro/rapidpro/issues/766

### CODE REVIEW

File: temba/msgs/models.py [hunk 1]
```diff
@@ -2383,22 +2383,11 @@ def create(cls, org, user, system_label=None, label=None, groups=(), start_date=
         return export
 
     def write_export(self):
-        from openpyxl import Workbook
+        from xlsxlite.writer import XLSXBook
```
Comment (rowanseymour) [inline temba/msgs/models.py]: doesn't need to be a relative import

**Comment by rowanseymour** [comment]
@norkans7 does this bring us closer to removing any excel related dependencies?

**Comment by norkans7** [comment]
we still rely on it in tests

**Comment by rowanseymour** [comment]
@norkans7 small thing but if we're only using openpyxl for testing then let's move that to the testing section in pip-requires.txt. That'll will help us remember why we still need it

**Pull request merged**